libs/cassandra-util/src/Cassandra.hs

```diff
@@ -1,4 +1,64 @@
+-- | The top import of our Cassandra utility library. Essentially, a "everyone who needs cassandra needs this" import.
 module Cassandra (module C) where
 
-import Cassandra.CQL  as C
-import Cassandra.Exec as C
+-- | pull in our extended wrapper of Database.CQL.Protocol
```

Comment:
Not sure Haddock will parse that (`-- |` is for declarations and an `import` is not a declaration)

```suggestion
-- pull in our extended wrapper of Database.CQL.Protocol
```